pb/summary/summary.proto

```diff
@@ -45,6 +47,73 @@ message FailingTestSummary {
   string fail_test_link = 13;
 }
 
+// Metrics about a specific test, i.e. passes, fails, total runs, etc.
+message TestInfo {
+  // The display name of the test, typically what is shown on the leftmost 
+  // column of each row of TestGrid
+  string display_name = 1;
+
+  // The total number of test runs not including runs failed due to infrastructure
+  // failures.
+  int32 total_non_infra_runs = 3;
+
+  // The number of passed test runs not including runs failed due to
+  // infrastructure failures.
+  int32 passed_non_infra_runs = 4;
+
+  // The number of failed test runs not including runs failed due to
+  // infrastructure failures.
+  int32 failed_non_infra_runs = 5;
+
+  // The number of failed test runs specifically due to infrastructure
+  // failures.
+  int32 failed_infra_runs = 6;
+
+  // The total number of all runs, including failures due to infrastructure
+  int32 total_runs_with_infra = 7;
+
+  // Any other type of runs not included above.
+  int32 other_runs = 8;
+
+  // The flakiness of the test, measured out of 100
+  float flakiness = 9;
+
+  // List of bug IDs for bugs associated with this test.
+  repeated string linked_bugs = 10;
+
+  // A summary of the infrastructure issues that make up the failed_infra_runs
+  // of the format "FAILURE_1 50% FAILURE_2 50%", where the sum of all
+  // percentages should be 100%
+  string infra_issues = 11;
```

Comment:
I think we want to drop the `infra_issues` field and have a better implementation?

Reply:
Okay; how about the all_infra_issues map in the HealthinessInfo message?

Reply:
Drop that as well? For now we can keep the `failed_infra_runs` counter and that should be good enough?

Reply:
Done